cabot/cabotapp/tasks.py

```diff
@@ -99,11 +99,7 @@ def clean_db(days_to_retain=7, batch_size=10000):
     InstanceStatusSnapshot.objects.filter(id__in=instance_snapshot_ids).delete()
 
     # If we reached the batch size on either we need to re-queue to continue cleaning up.
-    if (
-            result_count == batch_size or
-            service_snapshot_count == batch_size or
-            instance_snapshot_count == batch_size
-    ):
+    if result_count == batch_size or service_snapshot_count == batch_size or instance_snapshot_count == batch_size:
```

Comment:
Any reason for this style change? Do you mind keeping consistent unless there's a substantive change?

Reply:
the reason is pep8 test failing if that change is not present